tests/app-sync/run_seeds.go

```diff
@@ -0,0 +1,113 @@
+package appsync
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/celestiaorg/test-infra/testkit"
+	"github.com/celestiaorg/test-infra/testkit/appkit"
+
+	"github.com/testground/sdk-go/network"
+	"github.com/testground/sdk-go/run"
+	"github.com/testground/sdk-go/runtime"
+)
+
+// TODO(@Bidon15): seed nodes are not working as expected
+// Now this code is not used anywhere in test-plan/cases
+// More info to follow up: https://github.com/tendermint/tendermint/issues/9289
+func RunSeed(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*4)
+	defer cancel()
+
+	syncclient := initCtx.SyncClient
+	netclient := network.NewClient(syncclient, runenv)
+
+	netclient.MustWaitNetworkInitialized(ctx)
+
+	config := network.Config{
+		Network: "default",
+		Enable:  true,
+		Default: network.LinkShape{
+			// Latency:   100 * time.Millisecond,
+			Bandwidth: 4 << 26, // 256Mib
+		},
+		CallbackState: "network-configured",
+		RoutingPolicy: network.AllowAll,
+	}
+
+	config.IPv4 = runenv.TestSubnet
+
+	// using the assigned `GlobalSequencer` id per each of instance
+	// to fill in the last 2 octects of the new IP address for the instance
+	ipC := byte((initCtx.GlobalSeq >> 8) + 1)
+	ipD := byte(initCtx.GlobalSeq)
+	config.IPv4.IP = append(config.IPv4.IP[0:2:2], ipC, ipD)
+
+	err := netclient.ConfigureNetwork(ctx, &config)
+	if err != nil {
+		return err
+	}
+
+	home := fmt.Sprintf("/.celestia-app-%d", initCtx.GroupSeq)
+	runenv.RecordMessage(home)
+
+	cmd := appkit.New()
+
+	nodeId, err := cmd.GetNodeId(home)
+	if err != nil {
+		return err
+	}
+
+	err = <-syncclient.MustBarrier(ctx, testkit.FinalGenesisState, 1).C
+	if err != nil {
+		return err
+	}
```

Comment:
Should we remove this Barrier as well? Or do we use Barriers until they start causing problems? 😄

Reply:
Agreed. Removed 😅